data/motd/fwupd.conf

```diff
@@ -0,0 +1 @@
+u @systemd_unit_user@ - "Firmware update daemon" @localstatedir@/lib/fwupd
```

Comment:
I suggest this be renamed to make it more obvious what the file is.

Or if the file supports a comment in the syntax add that.

Since it's processed by meson we can still install into filesystem as fwupd.conf.

Reply:
Something like fwupd.sysusers? If so, fixed.